airbyte-integrations/base-python/.dockerignore

```diff
@@ -1,4 +1 @@
-*
-!Dockerfile
-!base.py
-!airbyte_protocol
+build
```

Comment:
Why not use the exclusion pattern for this?

Reply:
Good question :) we want to have a small context sent to docker for our core piece since we don't want to pull all the compiled and dist files but for connectors, having it it as an exclude list is very confusing to someone who tries to create a new connector since it becomes yet another place that you need to update.

Reply:
Alright sounds fine. We will have to be careful once these have python tests though.

Reply:
It is not a problem. they won't end up in the image just in the context used to build the image